content/resources/crownlabs_ssh/crownlabs_ssh.md

```diff
@@ -0,0 +1,97 @@
++++
+fragment = "content"
+#disabled = true
+date = "2021-11-12"
+weight = 100
+# title = "Connecting to CrownLabs with SSH"
++++
+
+## Introduction
+
+[**CrownLabs**](https://crownlabs.polito.it/) allows users access via SSH to their VMs using SSH directly on their machines.
+
+In this guide you are going to learn how to generate a key on the main operative systems (Windows, Linux, MacOS), how to load it on [**CrownLabs**](https://crownlabs.polito.it/) and log in your [**CrownLabs**](https://crownlabs.polito.it/)'VM using SSH from terminal.
+
+## How to generate an SSH key pair
+
+1. Open a terminal and run the following command to create a [**RSA**](<https://it.wikipedia.org/wiki/RSA_(crittografia)>) key pair :
```

Comment:
Would change the link to English documentation. Could also use `Ed25519` cryptography, but RSA is probably the default

Reply:
I think that the command could be changed into `ssh-keygen -t` followed by the key type. For instance, for 2048-bit RSA:

```bash
ssh-keygen -t rsa -b 2048
```

For ED25519:

```bash
ssh-keygen -t ed25519
```

Reply:
We decided to omit the `-t` parameter for simplicity